src/frontend/components/VideoPlayerConnected/VideoPlayerConnected.spec.tsx

```diff
@@ -135,17 +143,45 @@ describe('<VideoPlayerConnected />', () => {
               status: requestStatus.SUCCESS,
             },
           },
+          [modelName.THUMBNAIL]: {},
         },
       } as any;
       expect(mapStateToProps(state, props)).toEqual({
         jwt: 'foo',
         languageChoices,
+        thumbnail: null,
         timedtexttracks: {
           objects: [{ id: '42' }, { id: '84' }, { id: '168' }],
           status: requestStatus.SUCCESS,
         },
         video: { id: 42 },
       });
     });
+    it('retrieves thumbnail if present in the state', () => {
```

Comment:
`s/retrieves/fetches/` ?

Reply:
No, it doesn't fetch the API (if this is what you mean).

Reply:
No it's not what I meant: I find the "retrieve" verb not optimal.

Reply:
why ? we are looking if the resource exists in the state, retrieve is as appropriate as fetch IMO

Reply:
> ok